### PR TITLE
Adds medkits outside of all stations medbays

### PR DESCRIPTION
**Pull request by ghost**
[Changelogs]: # (Your PR should contain a detailed changelog of notable changes, titled and categorized appropriately. This includes, new features, sprites, sounds, balance changes, admin tools, map edits, removals, big refactors, config changes, hosting changes and important fixes. An example changelog has been provided below for you to edit. If you need additional help, read https://github.com/tgstation/tgstation/wiki/Changelogs)

:cl: Awesine
add: Added medkits outside of each stations medbays for public use.
/:cl:

[why]: # (Please add a short description [two lines down] of why you think these changes would benefit the game. If you can't justify it in words, it might not be worth adding.)

alternative to  #33686
also so greytiders don't break into medbay and steals kits

box:
https://i.imgur.com/fGr92lD.png
omega:
https://i.imgur.com/5i3e0Z7.png
pubby:
https://i.imgur.com/xRgUBKy.png
meta:
https://i.imgur.com/upPd8vC.png
delta
https://i.imgur.com/fRDC7Gp.png

### CODE REVIEW

**Comment by DaxDupont** [comment]
Don't make medbay's job even more boring please.

**Comment by awesine** [comment]
@DaxDupont 
how is this going to make it less boring?
it's going to make it less stressful when people break in to steal medkits when they need it/want it

**Comment by DaxDupont** [comment]
1. Because this will result in less people using medbay's services and this cauises less interdepartmental interaction
2. Medbay storage doesn't get broken into that much, at least not on bagil. As long there's a competent medbay. 
3. Breaking into storage also creates conflict which is what this game thrives on. 
4. Not only that, leaving even more medkits out creates power creep especially since non-regular medkits are more or less instaheals for their damage type.

**Comment by awesine** [comment]
>Medbay storage doesn't get broken into that much, at least not on bagil. As long there's a competent medbay.

you say this but it happens pretty much every round

>Breaking into storage also creates conflict which is what this game thrives on

apparently that's a bad thing according to xhuis and what not

>Not only that, leaving even more medkits out creates power creep especially since non-regular medkits are more or less instaheals for their damage type.

so do regular medkits

>Because this will result in less people using medbay's services and this cauises less interdepartmental interaction

this is already what medbay is

**Comment by DaxDupont** [comment]
1. I play medical doctor almost constantly, you're objectively wrong.
2. Also conflict is not a bad thing
3. Regular medkits require targeting and heal less in total vs a patch. They also contain less per damage type.
4. Wrong

**Comment by Xhuis** [comment]
Could you get some screenshots?

**Comment by awesine** [comment]
2. i said apparently.
1. I play medical doctor almost constantly, you're objectively wrong.
3. that may be true but not by a lot
4. wrong

**Comment by awesine** [comment]
screenshots coming soon

**Comment by epochayur** [comment]
As a frequent MD player, I'm cool with this. Keep your grubby greytide mitts off of storage

**Comment by awesine** [comment]
BOXSTATION:
https://i.imgur.com/Byizo0X.png

https://i.imgur.com/b6TCuzi.png

**Comment by epochayur** [comment]
@awesine is it just the one toxin kit or are there more stacked behind it? if the latter, consider spreading them out a bit to make it more obvious

**Comment by awesine** [comment]
METASTATION:
https://i.imgur.com/8n0OWsu.png

https://i.imgur.com/tJPiBgc.png

**Comment by awesine** [comment]
@epochayur 
it's stacked

**Comment by awesine** [comment]
PUBBY:
https://i.imgur.com/RNFRM4i.png

https://i.imgur.com/HHDbaD6.png

**Comment by Xhuis** [comment]
Having them all stacked on one tile is kind of sloppy and more of a bandaid fix to the issue; I'd say at least adjust their pixel offsets so they can actually be seen individually. Otherwise, I suppose it works.

**Comment by awesine** [comment]
I'm going to put half on the table next to it for each section in a minute

**Comment by awesine** [comment]
Omega:
https://i.imgur.com/InFS7Jy.png

https://i.imgur.com/t5MV8wA.png

**Comment by Potato-Masher** [comment]
If we are going in this direction, why not go the full length and just flat out _remove_ medbay? It'll be a greytide paradise: a giant box that's **filled** to the top with medkits of **every kind!** Now everyone can have medkits shoved up their arse because without them those _special snowflake tiders_ wouldn't have health they need to be a constant jackass and steal the armory "_because they can_"!

I mean honestly, those dumb medical doctors just tell me to stop breaking shit and don't heal **_me!_** They just call for shitcurity to arrest me for having _fun_, just because I needed **_my_** medkit just in case something happened to _**ME!!**_


Sorry for the harsh sarcasm, but this just makes the MD's job even more useless. Until we come up with some revolutionary idea to make the MD's job a little more engaging and stimulating, we shouldn't keep phasing them out. I just see too many downsides from my perspective to justify adding a free supply of medkits when it should be the chemist's job to make that free of basic drugs (and advanced ones for the medical staff).

**Comment by awesine** [comment]
@Penguaro 
this but unironically unironically

**Comment by Potato-Masher** [comment]
Yeah, to this day I still chuckle internally when a guy can take close to 5 solid bullets in the head and walks it off once someone slaps some powder in a patch on them, only to charge headlong into another salvo of LMG bullets to repeat the process. Something needs to be shifted. I have no real experience of Baymed, but I hear its pretty indepth, and that's why people like it.

**Comment by awesine** [comment]
Goonmod is also pretty good IMO, although some people will disagree because >goon

**Comment by awesine** [comment]
Images on the main DESC and it's ready to merge if it gets merged.

**Comment by awesine** [comment]
now completely ready

**Comment by awesine** [comment]
why did it failll reeeee

**Comment by MMMiracles** [comment]
why don't you just have everyone spawn with a medkit so we can make medbay even more irrelevant

**Comment by awesine** [comment]
@MMMiracles 
this but unironically

**Comment by Xhuis** [comment]
step_x and step_y don't work, use pixel_x and pixel_y instead.

**Comment by awesine** [comment]
@Xhuis 
i don't know how to change that

**Comment by MMMiracles** [comment]
you have to manually go into the map file via a text editor like Notepad++ and delete all instances of 'step_x' and 'step_y'

**Comment by awesine** [comment]
@MMMiracles 
is this what all mappers do damn

**Comment by Repukan** [comment]
Why not make everyone spawn with godmode on so you don't have to break into medbay to heal.

**Comment by awesine** [comment]
that but unironically

**Comment by awesine** [comment]
should be 100$% done

**Comment by kevinz000** [comment]
no we don't need so many free heals.

**Comment by Armhulen** [comment]
### FUCK YOU

**Comment by Armhulen** [comment]
## FUCK YOU

**Comment by Armhulen** [comment]
seriously

**Comment by Armhulen** [comment]
# FUCK YOU

**Comment by Armhulen** [comment]
look at this image HITLER, THAT'S TWO BURN, TWO TOX, TWO OXY AND FOUR NORMAL KITS (BRUTE AND BURN) 

WHAT THE FUCK

**Comment by Armhulen** [comment]
YOU ADDED A BRUTE KIT HERE TOO JESUS

**Comment by Armhulen** [comment]
fuck ow i'm just SO MAD MY GOD.

**Comment by Armhulen** [comment]
every map has an entire medbay storage outside i counted it up:
2 regular (brute + burn)
2 brute
2 fire
2 tox
2 oxy

**Comment by MrDoomBringer** [comment]
image

**Comment by awesine** [comment]
@Armhulen actually each storage has 3 inside lol

**Comment by awesine** [comment]
@MrDoomBringer 
HOLY SHIT THAT'S GOOD

**Comment by awesine** [comment]
@Armhulen 


anyway this is a good change that will promote less medbay looting

**Comment by awesine** [comment]
i am the best coder

make me maintainer @Cyberboss

**Comment by awesine** [comment]
i've already mapped lavaland gulag to have adv mining scanners
that is being pr'd next

watch out

the pr: part two

**Comment by Daturix** [comment]
Awesine just drop the pr. no one wants it and you're being a nuisance

**Comment by awesine** [comment]
@Daturix 
this but ironically

**Comment by Rinoka** [comment]
Are we done being Very Excited about making someone mad?

I get you're being kind of tongue-in-cheek and getting ~excited~ that people are getting mad at you, but also, flipping the "this but un/ironically" switch as it suits you just disappoints me.

You're really clearly trying to bait people into writing long-ass posts about why you're an asshole or something. Con... grats? I'm glad your first git pull has worked out so well for you.

Let's hope you don't end up being the assistant of the git community.

**Comment by awesine** [comment]
that but unironically

**Comment by Rinoka** [comment]
you realize that "that but unironically" is literally admitting to this being a bait PR, right? i was kind of hoping you'd at least try and keep this going a little longer by saying "but ironically" instead.

**Comment by awesine** [comment]
that but this isn't actually a bait PR and i really want this

**Comment by jughu** [comment]
I dont getting auctall patients is a rare thing and then im the darn EMT in that situation

**Comment by awesine** [comment]
english pls

**Comment by Rinoka** [comment]
if this isn't a bait pr: tell medics to do it for you. make them dump out basic medkits in the main area. or just rely on the chemist to do their job.

it might be a "QoL" thing for you but there's still literally nothing stopping people from just running over and taking these. this is part of why some areas have job-specific medikits already provided. this was already a thing, as i recall, way earlier in the game cycle and stopped being a thing because it was terrible and resulted in even more shitty powergaming.

**Comment by Rinoka** [comment]
also they said "i don't want this. getting actual patients is a rare thing, and then when I do get patients i'm the fucking EMT." aka, nobody comes to medbay to be healed.

**Comment by Armhulen** [comment]
@Rinoka i wouldn't bother they're laughing about baiting people on discord so

**Comment by jughu** [comment]
CLOSE

**Comment by Rinoka** [comment]
i continue to be 99% accurate at translating random text

also i'm well aware they're in it for the B8 but if they're having a good time, i don't mind baiting the baiter. power to them.

**Comment by awesine** [comment]
i'm not in for the bait though the actual PR itself is serious
these comments are not

**Comment by Rinoka** [comment]
god, i hope you're being serious, even considering. that would be the most amazing thing i'd ever watch happen - someone responds to all criticism with trolling but doesn't understand why their Legit Coder Idea doesn't get implemented.

maybe kevin set a bad example with techwebs.

**Comment by awesine** [comment]
this but ironically

**Comment by awesine** [comment]
except the kevinz part

techwebs is bad

**Comment by Rinoka** [comment]
it's a pretty apt comparison. this change is bad. so was techwebs.

**Comment by awesine** [comment]
except my change might actually make the rounds more fun

**Comment by Rinoka** [comment]
this change existed in some previous versions a while ago. powergamers stole the public ones most of the time, and then once they were out, shit got broken into anyway. this was pre-chemistry; chemistry was a fix to that problem. "might" is a bit of a stretch.

**Comment by awesine** [comment]
now that chemistry is here it is a nice balance

**Comment by Rinoka** [comment]
powergamers are still going to steal them. there's no challenge or restrictions to it, and health kits are some of the more limited but robust items in the game for healing if the chemist has already fucked off. all it takes is 12 greedy powergamers (assuming they just grab one, even!) and suddenly all you've done is give more power to people who desperately love having it.

if you prefer it this way, you could try and convince someone to testmerge this. you'd probably have to actually take the pr seriously and spend less time celebrating baiting people on the internet, but you could do that. but i think that once people realize they're perma-there, you'd see a lot of powergaming surge back.

but i think you might be better off dropping this. you've clearly had your fun, and if you want to have more fun baiting people into giving a shit, be my guest. but the merge itself is a trip backwards for the git, so if you want it, you might need to come up with a better reason than "but what if it works".

**Comment by awesine** [comment]
it's already worken though in practice for like
3+ years

have you seen goonstation?

**Comment by jughu** [comment]
>goonstation

2-3 medkits brute/burn they dont use normal medkits with bruisepacks and ointment like we do ((which is the majority off the kits outside of medbay)

Goonstation has a public medivendor which you have to pay for so why are you trying to get goon involved into this lol they dont even use normal medkits because they only use brute/burn/toxin/suffocation/trauma

**Comment by Kraseo** [comment]
we didn't kick one goofball out only to have another one come in like a fucking revolving door

**Comment by Magikarpet** [comment]
give medical self serve sleepers as well and remove medical doctors as a role

**Comment by Ratvar** [comment]
Add 5 tasers into every security checkpoint so antags won't need to break into armory

**Comment by DaxDupont** [comment]
Why are there 17 commits. 

I am 100% convinced this is a low effort meme PR now.

**Comment by WJohn** [comment]
This would basically invalidate medbay. That's a fucking assload of medkits jesus christ. Who would ever need doctors when you have basically as much medical equipment just outside medbay it as in the secure medical storage?

File: _maps/map_files/BoxStation/BoxStation.dmm
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: dirty var

File: _maps/map_files/BoxStation/BoxStation.dmm
Comment (ShizCalev) [inline _maps/map_files/BoxStation/BoxStation.dmm]: dirty var

File: _maps/map_files/MetaStation/MetaStation.dmm
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: dirty var

File: _maps/map_files/MetaStation/MetaStation.dmm
Comment (ShizCalev) [inline _maps/map_files/MetaStation/MetaStation.dmm]: dirty var

File: _maps/map_files/OmegaStation/OmegaStation.dmm
Comment (ShizCalev) [inline _maps/map_files/OmegaStation/OmegaStation.dmm]: dirty var

File: _maps/map_files/OmegaStation/OmegaStation.dmm
Comment (ShizCalev) [inline _maps/map_files/OmegaStation/OmegaStation.dmm]: dirty var

File: _maps/map_files/PubbyStation/PubbyStation.dmm
Comment (ShizCalev) [inline _maps/map_files/PubbyStation/PubbyStation.dmm]: dirty var

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Comment by Aloraydrel** [comment]
I am about 90% sure this is ironic.

**Comment by DaedalusGame** [comment]
he used the github interface instead of doing shit like a normal person

File: _maps/map_files/BoxStation/BoxStation.dmm
Comment (Xhuis) [reply _maps/map_files/BoxStation/BoxStation.dmm]: Show me your naughty, dirty vars, Shiz~

**Comment by awesine** [comment]
@WJohn 
actually medbay has 3 medkits each not 2, making them have 15 medkits not 10

**Comment by Magikarpet** [comment]
awesine it did actually exist the syndicate have self-serve sleepers aka the red sleepers just change the icon state and steal the code

File: _maps/map_files/PubbyStation/PubbyStation.dmm
Comment (MrStonedOne) [reply _maps/map_files/PubbyStation/PubbyStation.dmm]: To answer this question, This is what happens if you use the nudge-* commands in the map editor without enabling nudge as pixels.

**Pull request closed without merging**
